ipapython/ntpmethods.py

```diff
@@ -0,0 +1,243 @@
+#
+# Copyright (C) 2018  FreeIPA Contributors see COPYING for license
+#
+from __future__ import absolute_import
+
+import shutil
+import re
+import sys
+
+# pylint: disable=import-error,ipa-forbidden-import
+from ipaclient.install import ipadiscovery  # pylint: disable=E0401,E0611
```

Comment:
/usr/lib/python3.6/site-packages/ipaclient/install/ipadiscovery.py:22: DeprecationWarning: ipaclient.install.ipadiscovery is deprecated, use ipaclient.discovery

Reply:
try
```python
from ipaclient import discovery
```